code/modules/antagonists/eldritch_cult/knowledge.dm/rust_lore.dm

```diff
@@ -0,0 +1,248 @@
+/datum/eldritch_knowledge/base_rust
+	name = "Blacksmith's Tale"
+	desc = "Opens up the path of rust to you. Allows you to transmute a knife with any trash item into a Rusty Blade."
+	gain_text = "Let me tell you a story, blacksmith said as he glazed into his rusty blade."
+	banned_knowledge = list(/datum/eldritch_knowledge/base_ash,/datum/eldritch_knowledge/base_flesh,/datum/eldritch_knowledge/ash_final,/datum/eldritch_knowledge/flesh_final)
+	next_knowledge = list(/datum/eldritch_knowledge/rust_fist)
+	required_atoms = list(/obj/item/kitchen/knife,/obj/item/trash)
+	result_atoms = list(/obj/item/melee/sickly_blade/rust)
+	cost = 1
+	route = "Rust"
+
+/datum/eldritch_knowledge/rust_fist
+	name = "Grasp of rust"
+	desc = "Empowers your mansus grasp to deal 500 damage to non-living matter and rust any turf it touches. Destroys already rusted turfs."
+	gain_text = "Rust grows on the ceiling of the mansus."
+	cost = 1
+	next_knowledge = list(/datum/eldritch_knowledge/rust_regen)
+	var/rust_force = 500
+	var/static/list/blacklisted_turfs = typecacheof(list(/turf/closed,/turf/open/space,/turf/open/lava,/turf/open/chasm,/turf/open/floor/plating/rust))
+	route = "Rust"
+
+/datum/eldritch_knowledge/rust_fist/mansus_grasp_act(atom/target, mob/user, proximity_flag, click_parameters)
+	. = ..()
+	if(ishumanbasic(target))
+		var/mob/living/carbon/human/H = target
+		var/datum/status_effect/eldritch/E = H.has_status_effect(/datum/status_effect/eldritch/rust) || H.has_status_effect(/datum/status_effect/eldritch/ash) || H.has_status_effect(/datum/status_effect/eldritch/flesh)
+		if(E)
+			E.on_effect()
+			H.bleed_rate = min(H.bleed_rate + 4, 8)
+			for(var/atom/X in range(1,H))
+				if(prob(75))
+					conv_area(target)
+			return
```

Comment:
so having a mark on the target gives you a 25% chance to NOT conv_area() on their turf?